README.md

```diff
@@ -104,12 +115,234 @@ Ansible based automation
 
 You can use the Ansible playbooks and roles at <https://github.com/rh-nfv-int/nfv-example-cnf-deploy> to automate the use of the Example CNF.
 
+Load balancer mode vs. direct mode
+------------------------
+
+There are two different ways of deploying example-cnf, depending on the `enable_lb` flag (default to `true`):
+
+- Load balancer mode (`enable_lb: true`):
+    - Represents the behavior described above, with a TestPMD instance acting as load balancer between TRex and the CNF Applications.
+- Direct mode (`enable_lb: false`):
+    - The TestPMD instance is not used, then testpmd-lb-operator is not deployed.
```

Comment:
"The TestPMD instance is not used, then testpmd-lb-operator is not deployed" - do we speak here about loadbalancer or about loadbalancer-lb? Right above, both are mixed up

Reply:
I made a mistake here, I will fix that